onboarding/urls.py

```diff
@@ -17,7 +16,7 @@
 # base
 urlpatterns = [
     path('', views.index, name='index'),
-    url('signup', core_views.signup, name='signup'),
+    url('signup', views.signup, name='signup'),
```

Comment:
Good idea :+1:  :-)
Maybe we should wait what author would say about that - lets give for Michał some time to see that.